user-interface/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectContentComponent.java

```diff
@@ -29,13 +35,39 @@ public class ProjectContentComponent extends Div {
   private static final long serialVersionUID = -1061134126086910532L;
   private final ProjectDetailsComponent projectDetailsComponent;
 
+  private final ProjectPurchaseService projectPurchaseService;
+  private Context context;
+
   public ProjectContentComponent(
-      @Autowired ProjectDetailsComponent projectDetailsComponent) {
+      @Autowired ProjectDetailsComponent projectDetailsComponent,
+      @Autowired ProjectPurchaseService projectPurchaseService) {
     Objects.requireNonNull(projectDetailsComponent);
+    this.projectPurchaseService = Objects.requireNonNull(projectPurchaseService);
+    UploadPurchaseDialog uploadPurchaseDialog = new UploadPurchaseDialog();
```

Comment:
NIT: I'd suggest to re-create the dialog every time it is opened. This way you can be certain that it is refreshed.

Reply:
agree, but since this is not the final implementation and just a showcase, this can be addressed in the final implementation